components/accordion/stories/accordion.stories.js

```diff
@@ -8,9 +8,31 @@ export default {
   component: "Accordion",
   argTypes: {
     items: { table: { disable: true } },
+    size: {
+      name: "Size",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["s", "m", "l", "xl"],
+      control: "select"
+    },
+    layout: {
+      name: "Layout",
```

Comment:
I think "layout" should probably be renamed to "density". Other components like Table are using this terminology for spacious and compact. I don't think this was specifically mentioned anywhere for Accordion.